easybuild/tools/filetools.py

```diff
@@ -1166,12 +1167,15 @@ def rmtree2(path, n=3):
 
 def find_backup_name_candidate(src_file):
     """Returns a non-existing file to be used as destination for backup files"""
-    dst_file = src_file
 
-    i = 0
+    # e.g. 20170817234510 on Aug 17th 2017 at 23:45:10
+    timestamp = datetime.datetime.now()
+    dst_file = '%s_%s' % (src_file, timestamp.strftime('%Y%m%d%H%M%S'))
     while os.path.exists(dst_file):
```

Comment:
Fair point, even though the "rapid succession" would have to be very rapid for falling in the same second. But ok, better safe than sorry, and it doesn't hurt.